src/AmbientSounds.Uwp/Views/ScreensaverPage.xaml.cs

```diff
@@ -232,5 +245,39 @@ private void OnToggleFullscreen(object sender, RoutedEventArgs e)
                 });
             }
         }
+
+        private void SetTimer()
+        {
+            InactiveTimer = new Timer(MilisecondsToHide);
```

Comment:
This is not correct. I specifically suggested to use a `DispatcherQueueTimer` to avoid this.
You should do the following:
- Get the current queue in the constructor and store it in a field
- When creating the timer, use this: [DispatcherQueue.CreateTimer](https://docs.microsoft.com/en-us/uwp/api/windows.system.dispatcherqueue.createtimer)
- Use that timer for everything. That will already dispatch on the correct thread.